py/head.py

```diff
@@ -2,18 +2,56 @@
 from dominate.tags import *
 
 def _head():
-    """Returns the generated HTML header. TODO: edit this description
-
-    TODO: A description here, according to Google's Python style guide https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
+    """This function generates a header using table elements and returns it as HTML. The header consists of the banner logo
+    (lightbulbs and title) and the social media handles with their icons. The banner links to the main TSE website and each
+    social media handle links to their respective accounts. This section of the newsletter should not need to be changed
+    since no new information needs to be added. DO NOT EDIT!
 
     Args:
-        TODO
+        None
 
     Returns:
-        TODO
+        The generated HTML header. The header consists of table elements split into two rows. The top row contains the banner and the
```

Comment:
I don't think this is true. Aren't we supposed to be returning a type of object and not simply just an HTML header? Talk more about what exactly the type of the object we are returning here.